playbooks/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+- name: Install apt package dependencies.
+  apt:
+    name: apt-transport-https
+    state: present
+
+- name: Add Elastic's apt key
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch
+    state: present
+
+- name: Add Elastic's apt repository
+  apt_repository:
+    repo: 'deb https://artifacts.elastic.co/packages/6.x/apt stable main'
```

Comment:
@smarnach Btw, I think this conflicts with the one upstream installs: https://github.com/edx/configuration/blob/master/playbooks/roles/elasticsearch/defaults/main.yml#L7

So if there is an upgrade run on the server it will probably upgrade Elasticsearch too if the service is local, is that right? Is this something to worry about?

Reply:
Yeah, we are running Elasticsearch mostly directly on the app servers, and we want to use version 1.5, because it's the only version that works.  If this repository adds a package with the name `elasticsearch` with a newer version than 1.5, it may be updated.  We could work around this by decreasing the priority of the repo we add in `/etc/apt/priorities` to a value that ensures that packages from that repo never override packages from other repos.

Reply:
Yeah I just tested it, and it indeed got updated to 6.2.4. But it was the correct version of 1.5 after running all playbooks, so as long as no one "accidentally" runs an `apt-get upgrade` on any appserver manually, things should be alright. But we should definitely pin it anyway. I'll add a task for that.

Reply:
After downgrading back to 1.5 and adding the priority (template in an upcoming commit), I get:

```
The following packages have been kept back:
  elasticsearch git git-man linux-headers-generic linux-headers-virtual linux-image-virtual linux-virtual rabbitmq-server
0 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
```

So it should be safe now.